### DISCUSSION TITLE
L1 loss in quantile regression forest

### DISCUSSION THREAD

**Discussion by yvc5154** [kogalur/randomForestSRC#433] [Q&A]
Hello,

I am looking the manual of randomForestSRC. In rfsrc, it says splitrule="quantile.regr": quantile regression splitting via the "check-loss" function. When I look at quantreg.rfsrc, I cannot see any explanation about check-loss (L1 loss) function. I am confused because of two different explanations. Could you clarify this?

Sincerely,

Youngjoo Cho
[randomForestSRC.pdf](https://github.com/user-attachments/files/15833063/randomForestSRC.pdf)

**Comment by ishwaran** [comment]
The default splitting uses the quantile loss  (pinball loss) which is defined by the check-loss function.  Basically this means that the splitting rule is designed so that the optimal solution is the quantile of interest.  This is similar to say regression where the splitting rule is weighted variance and the optimal solution is the conditional mean.  In other words, the loss used is designed so that the optimal value is the target that we are after.
